src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3553,16 +3554,17 @@ void ApplicationManagerImpl::ProcessApp(const uint32_t app_id,
                       << std::boolalpha << end_streaming_by_hmi_level);
     EndNaviServices(app_id);
   }
+  LOG4CXX_TRACE(logger_, "No actions required for app " << app_id);
```

Comment:
@mkorniichuk you should also add `return` for the last if statement to avoid  double logging

Reply:
@AKalinich-Luxoft fixed in 9a3bfe0aef2ba3121af594707ec1728d5480f057